core/conversion/converters/converter_util.cpp

```diff
@@ -199,6 +199,60 @@ nvinfer1::ITensor* tensor_to_const(ConversionCtx* ctx, at::Tensor t, const std::
   return out;
 }
 
+nvinfer1::ITensor* toITensor(ConversionCtx* ctx, const torch::jit::Node* n, at::Tensor* input) {
+
+    auto weights = Weights(ctx, *input);
+    // IConstantLayer to convert indices from Weights to ITensor
+    auto const_layer = ctx->net->addConstant(weights.shape, weights.data); // shouln't use constant
+    TORCHTRT_CHECK(const_layer, "Unable to create constant layer from node: " << *n);
+    auto const_out = const_layer->getOutput(0);
+    return const_out;
+}
+
+// clamp x to [lower_bound, upper_bound]
+nvinfer1::ITensor* clamp(ConversionCtx* ctx, const torch::jit::Node* n, nvinfer1::ITensor* x,
+                        nvinfer1::ITensor* lower_bound, nvinfer1::ITensor* upper_bound) {
+  auto max_layer = ctx->net->addElementWise(*x, *lower_bound, nvinfer1::ElementWiseOperation::kMAX);
+  auto max_itensor = max_layer->getOutput(0);
+  auto min_layer = ctx->net->addElementWise(*max_itensor, *upper_bound, nvinfer1::ElementWiseOperation::kMIN);
+  auto min_itensor = min_layer->getOutput(0);
+  return min_itensor;
+}
+
+// return indices < 0 ? inputDims + indices : indices
+nvinfer1::ITensor* bump_if_negtive(ConversionCtx* ctx, const torch::jit::Node* n, nvinfer1::ITensor* input_dim,
```

Comment:
This needs a better name, maybe something like `normalize_indices` if I understand what this is doing (taking negative dimensions and wrapping them around). Also how would this work with dynamic shape?

Reply:
Slice layer need start/size ShapeTensor as input. If there are negtive indices in the shapeTensor, we should add some offet to them to make sure the value are positive.